<?xml version="1.0"?>
<robot
    xmlns:xacro="http://www.ros.org/wiki/xacro" name="beetle" >
  <xacro:arg name="robot_name" default="beetle" />

  <xacro:include filename="$(find beetle)/robots/beetle-art.urdf.xacro" />

  <!-- gazebo plugin for default controller and sensors -->
  <xacro:include filename="$(find aerial_robot_simulation)/xacro/spinal.gazebo.xacro" />
  <xacro:gazebo_spinal robot_name="$(arg robot_name)" mag_frame="fc" />
</robot>
